pkg/kubelet/server/server.go

```diff
@@ -417,6 +419,20 @@ func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
 	}
 }
 
+// InstallDebuggingDisabledHandlers registers the HTTP request patterns that provide better error message
+func (s *Server) InstallDebuggingDisabledHandlers() {
+	glog.Infof("Adding debug disabled handlers to kubelet server.")
+
+	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		http.Error(w, "Server endpoints for log collection and local running of containers and commands are disabled.", http.StatusMethodNotAllowed)
```

Comment:
I would reword this to: `Debug endpoints are disabled.`

Reply:
Done.